### PR TITLE
adding HTC user guide to yin-chen-patch-1

### PR DESCRIPTION
**Pull request by YinChenEGI**
# Summary 

----

**Related issue :*v0.1*

### CODE REVIEW

File: content/en/users/htc/_index.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+weight: 20
+description: >
+  EGI High Throughput Compute (HTC)
+---
+EGI High Throughput Compute service is provided by a distributed network of computing centres, accessible via a standard interface and membership of a Virtual Organisation.
+### What is High Throughput Compute? 
+High Throughput Compute (HTC) can be described as a use case for an infrastructure that includes a big number of tasks (that could be data analysis tasks, also computing jobs for analysing data or other similar tasks) that are submitted in big numbers and loosely coupled. They can be executed individually and don’t need to interact with each other (too much). Users usually submit these tasks to the infrastructure in a big number of them and once the jobs have been scheduled and executed in the distributed resources of the infrastructure, the user can collect the output from the services that have executed their jobs.
+
+### The EGI HTC Infrastructure
+EGI HTC infrastructure is the federation of GRID resources provided by EGI providers, that aims to share and federate in a secure way distributed IT resources as part of the EGI infrastructure. It comprises
```
Comment (enolfc) [inline content/en/users/htc/_index.md:14]: GRID is probably meaningless to new users

File: content/en/users/htc/_index.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+- Computer resources -- execution environment of computing tasks in clusters distributing in many different resource centers over Europe and outside Europe ;
```
Comment (enolfc) [inline content/en/users/htc/_index.md:15]:
```suggestion
- Computer resources -- execution environment of computing tasks in clusters distributed in many different resource centers over Europe and outside Europe ; 
```

File: content/en/users/htc/_index.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+High Throughput Compute (HTC) can be described as a use case for an infrastructure that includes a big number of tasks (that could be data analysis tasks, also computing jobs for analysing data or other similar tasks) that are submitted in big numbers and loosely coupled. They can be executed individually and don’t need to interact with each other (too much). Users usually submit these tasks to the infrastructure in a big number of them and once the jobs have been scheduled and executed in the distributed resources of the infrastructure, the user can collect the output from the services that have executed their jobs.
+
+### The EGI HTC Infrastructure
+EGI HTC infrastructure is the federation of GRID resources provided by EGI providers, that aims to share and federate in a secure way distributed IT resources as part of the EGI infrastructure. It comprises
+- Computer resources -- execution environment of computing tasks in clusters distributing in many different resource centers over Europe and outside Europe ; 
+- Data infrastructure -- storage servers where users can upload and their data/files in a distributed manner in the different resource centers
```
Comment (enolfc) [inline content/en/users/htc/_index.md:16]:
```suggestion
- Data infrastructure -- storage servers where users can upload and download their data/files in a distributed manner in the different resource centers 
```

File: content/en/users/htc/_index.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+
+### The EGI HTC Infrastructure
+EGI HTC infrastructure is the federation of GRID resources provided by EGI providers, that aims to share and federate in a secure way distributed IT resources as part of the EGI infrastructure. It comprises
+- Computer resources -- execution environment of computing tasks in clusters distributing in many different resource centers over Europe and outside Europe ; 
+- Data infrastructure -- storage servers where users can upload and their data/files in a distributed manner in the different resource centers 
+- Federated operations -- a set of federated operations that are constituted of global tasks (central activities/ services, e.g., AAI, accounting, helpdesk) that are needed to federate the heterogeneous resources from different resource centers and their operation activities that are carried out by the different National Grid Initiative (NGIs).
```
Comment (enolfc) [inline content/en/users/htc/_index.md:17]: NGIs is no longer an acronym

File: content/en/users/htc/_index.md [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+
+### The EGI HTC Infrastructure
+EGI HTC infrastructure is the federation of GRID resources provided by EGI providers, that aims to share and federate in a secure way distributed IT resources as part of the EGI infrastructure. It comprises
+- Computer resources -- execution environment of computing tasks in clusters distributing in many different resource centers over Europe and outside Europe ; 
+- Data infrastructure -- storage servers where users can upload and their data/files in a distributed manner in the different resource centers 
+- Federated operations -- a set of federated operations that are constituted of global tasks (central activities/ services, e.g., AAI, accounting, helpdesk) that are needed to federate the heterogeneous resources from different resource centers and their operation activities that are carried out by the different National Grid Initiative (NGIs). 
+- User supports --  NGIs also carry out user support. EGI.eu provides the central user support and coordinates NGIs’ support activities.
```
Comment (enolfc) [inline content/en/users/htc/_index.md:18]:
```suggestion
- User support --  NGIs also carry out user support. EGI.eu provides the central user support and coordinates NGIs’ support activities.
```

**Review by enolfc — COMMENTED** [review]
Thanks @YinChenEGI! Nice to have HTC on board also.

I think we need to have some discussion on how the documentation should be organised, I feel there are several parts here that should be moved to a common section.

**Comment by YinChenEGI** [comment]
Hi Enol

Thanks very much for your kind message :)

It's my first try and I'm still in the process of checking with people
about the current status of HTC on various aspects.

Sure -- we should discuss the overall structure and overlapping contents ...

Also, got feedback from Baptiste -- he doesn't like long line :( -- so will
be a lot of changes/updates

best

Yin

**Comment by andrea-manzi** [comment]
Hi Yin,
thanks a lot for the PR.
Recently we discussed a reorganization of the Online storage service and we have proposed to move the Grid Storage offering under HTC. Therefore i would like to include in the HTC guide also info on Grid Storage, maybe extending the the section that you have already provided, Let me know what you and @CatalinCondurache think about this
thanks
Andrea

**Comment by gwarf** [comment]
@YinChenEGI for helping with fixing the line length in VS Code, you can use the rewrap extension: https://marketplace.visualstudio.com/items?itemName=stkb.rewrap

* In VScode press F1
* Look for install extensions
* Look for rewrap

Once installed you can select a paragraph and press `Options + q` (on Mac) to get it wrapped.

Here is some doc and also info about some other extensions may be worth to try (like markdown lint): https://code.visualstudio.com/Docs/languages/markdown

**Comment by YinChenEGI** [comment]
Dear Andrea

It's certainly no problem for me. We can have a meeting, together with
Catalin -- I also want to take the chance to ask you about the current
status of HTC in various aspects.

Looking forward to talking to you soon

Yin

**Comment by YinChenEGI** [comment]
Hi Baptiste

This works well --thanks very much!

best regards

Yin

File: content/en/users/htc/_index.md [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+
+### The EGI HTC Infrastructure
+EGI HTC infrastructure is the federation of GRID resources provided by EGI providers, that aims to share and federate in a secure way distributed IT resources as part of the EGI infrastructure. It comprises
+- Computer resources -- execution environment of computing tasks in clusters distributed in many different resource centers over Europe and outside Europe ; 
+- Data infrastructure -- storage servers where users can upload and download their data/files in a distributed manner in the different resource centers 
+- Federated operations -- a set of federated operations that are constituted of global tasks (central activities/ services, e.g., AAI, accounting, helpdesk) that are needed to federate the heterogeneous resources from different resource centers and their operation activities that are carried out by the different National Grid Initiative (NGIs).
```
Comment (enolfc) [inline content/en/users/htc/_index.md:17]:
```suggestion
- Federated operations -- a set of federated operations that are constituted of global tasks (central activities/ services, e.g., AAI, accounting, helpdesk) that are needed to federate the heterogeneous resources from different resource centers and their operation activities that are carried out by the different NGIs. 
```

**Review by enolfc — APPROVED** [review]
I think we can go and merge now and then tackle what I think are the main issues:
1. Move AAI part to the Getting Started/Check-in section as much as possible
2. Move the workload manager to its own section.
3. Decide how to deal with storage

Regarding (2), I have some work in progress already that touches also aod, so will come in a subsequent PR
(1) and (3) will be fixed as we improve all the documents during this month

**Comment by YinChenEGI** [comment]
Hi Enol
Just let you know:
A user document about Workload manager is under preparation at:
https://docs.google.com/document/d/1x-hYoyTlJQI2Nx1ud53NEnrrhUla1TyC6WhKGD880FI/edit#heading=h.p4e7nf1781i6

I invited Cataline to work on it together

best

Yin

**Review by enolfc — APPROVED** [review]

**Pull request merged**